run.sh

```diff
@@ -16,14 +16,20 @@ while [ -n "$1" ]; do
 done
 
 SDC_HOME=${SDC_HOME:-$(mktemp -d)}
+DB_HOME=$SDC_HOME/data
+LOGS_HOME=$SDC_HOME/logs
+
+export SDC_HOME DB_HOME LOGS_HOME
+
+mkdir -p $DB_HOME
+mkdir -p $LOGS_HOME
+chmod 0700 $SDC_HOME
+chmod 0700 $DB_HOME
+chmod 0700 $LOGS_HOME
 
 echo "Running app with home directory: $SDC_HOME"
 
 # create the database for local testing
-
-python - << EOF
-from securedrop_client.models import Base, make_engine
-Base.metadata.create_all(make_engine("$SDC_HOME"))
-EOF
+./createdb.py $DB_HOME
```

Comment:
I think it's fine to leave this as a here-doc since `run.sh` is a dev-only tool to help people with laughing the app while iterating. Additionally, this doesn't solve the "bootstrap at first run" ticket.

Reply:
Bootstrapping in production is handled by the `securedrop-client` script.